app/main.py

```
@@ -86,6 +88,17 @@ def create_application() -> FastAPI:
         allow_headers=["*"],
     )
     
+    # Add correlation ID handling (must be first to set correlation ID)
+    log_config = LogConfig()
+    app.add_middleware(
+        CorrelationIdMiddleware,
+        header_name=log_config.correlation_id_header,
+        generate_id_if_missing=True
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)